content/shell/browser/shell_content_browser_client.cc

```diff
@@ -625,6 +625,14 @@ base::FilePath ShellContentBrowserClient::GetFirstPartySetsDirectory() {
 }
 
 std::string ShellContentBrowserClient::GetUserAgent() {
+  base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
```

Comment:
This code should live under /cobalt somewhere, not as a direct modification to upstream. If that's not viable at all, it should be either run-time gated with a `cobalt` conditional ( preferable ) or runtime-gated if #ifdef

Reply:
Ok. I can subclass `ShellMainDelegate` to an implementation that replaces `CreateContentBrowserClient` with one that instantiates a subclass of `ShellContentBrowserClient` that returns a Cobalt string for `GetUserAgent`.